db/migrate/20201210151635_create_do_not_disturb_timings.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateDoNotDisturbTimings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :do_not_disturb_timings do |t|
+      t.integer :user_id, null: false
+      t.datetime :starts_at, null: false
+      t.datetime :ends_at, null: false
+    end
+    add_index :do_not_disturb_timings, [:user_id], unique: false
```

Comment:
I noted this above but you will probably want to index on the dates too if you are querying them.